src/uwtools/drivers/forecast.py

```diff
@@ -231,14 +231,47 @@ def _define_boundary_files(self) -> Dict[str, str]:
 
         return boundary_files
 
+    def _prepare_and_run_batch_script(self, run_directory: str) -> bool:
```

Comment:
I learned from _Clean Code_ that the presence of the word "and" in a function/method name might indicate that the function/method could usefully be split into two. These do seem like potentially two distinct operations that don't necessarily belong together: It seems like it might be useful to create a batch script without submitting it, or to submit a batch script that we already have for whatever reason. I'm ok with not changing this in this PR, but it does seem like a split might make sense.